src/main/scala/com/linagora/openpaas/gatling/addressbook/ContactSteps.scala

```diff
@@ -31,10 +31,13 @@ object ContactSteps {
       .check(status is 201))
   }
 
-  def listContactsFromAddressBook(addressBookLink: String): HttpRequestBuilder =
-    withAuth(http(s"listContactsFromAddressBook")
-      .get(s"/dav/api$addressBookLink?limit=20&offset=0&sort=fn")
-      .check(status in (200, 304)))
+  def listContactFromAddressBookQuery(addressBookLink: String): HttpRequestBuilder = http(s"listContactsFromAddressBook")
```

Comment:
why not more withAuth() here?

Reply:
oh I think I got it, can you just add the the `private` modifier like in other scenarios?